applications/users/controllers.py

```diff
@@ -103,6 +104,10 @@ def is_user_authorized_to_update_group(request, username, group_id):
 def get_user(request, email):
 	return db.get_user(request.db_session, email) if email is not None else None
 
+def get_user_profile(request, email):
```

Comment:
Can we merge `get_user` and `get_user_profile` ? There is only one line difference. Although auth token functionality wouldn't be used by other callers of `get_user` function, there is no need to create a new controller function. What do you think ?